package.json

```diff
@@ -24,6 +24,7 @@
   },
   "homepage": "https://github.com/zooniverse/liberating-the-liberator",
   "dependencies": {
+    "counterpart": "^0.18.1",
```

Comment:
What's this doing here? Isn't this a library for handling multiple languages in our UI?

Reply:
I included this because we're saving language information in the metadata of a classification with `counterpart.getLocale()`. With this PR, I am constructing metadata similar to how it is set on PFE.